components/dashboard/src/projects/ProjectListItem.tsx

```diff
@@ -35,26 +36,24 @@ export const ProjectListItem: FunctionComponent<ProjectListItemProps> = ({ proje
                         <ProjectLink project={project} />
                         <span className="flex-grow" />
                         <div className="justify-end">
-                            <ContextMenu
-                                menuEntries={[
-                                    {
-                                        title: "New Workspace",
-                                        href: gitpodHostUrl.withContext(`${project.cloneUrl}`).toString(),
-                                        separator: true,
-                                    },
-                                    {
-                                        title: "Settings",
-                                        link: `/projects/${project.id}/settings`,
-                                        separator: true,
-                                    },
-                                    {
-                                        title: "Remove Project",
-                                        customFontStyle:
-                                            "text-red-600 dark:text-red-400 hover:text-red-800 dark:hover:text-red-300",
-                                        onClick: () => setShowRemoveModal(true),
-                                    },
-                                ]}
-                            />
+                            <DropdownActions>
+                                <DropdownMenuItem asChild>
+                                    <a href={gitpodHostUrl.withContext(`${project.cloneUrl}`).toString()}>
```

Comment:
I tried using links, but the problem was that they were pointing me to a relative path instead of an absolute link. I was being sent to a URL like `https://gitpod-preview-env.com/https://gitpod-preview-env.com/start_workspace`, and couldn't figure out if there is a way to disable that behavior of forcing relative links.